scripts/mina-local-network/mina-local-network.sh

```diff
@@ -562,20 +585,48 @@ if ${VALUE_TRANSFERS}; then
     sleep 1
```

Comment:
You also need to check the `Whale` daemon availability since zkApps txns also depend on at least one `Whale`.

Reply:
I didn't use `Whale` daemon's functionality. I just need it's key.